jupyter_collaboration/events/session.yaml

```diff
@@ -0,0 +1,50 @@
+"$id": https://events.jupyter.org/jupyter_server/jupyter_collaboration/v1
+version: 1
+title: Events during collaborative sessions
+personal-data: true
+description: |
+  Record the event that happen in the backend during a collaborative session.
+type: object
+required:
+  - room
+  - path
+  - action
+properties:
+  room:
+    type: string
+    description: |
+      Room Id. Usually composed by the file type, format and Id.
+  path:
+    type: string
+    description: |
+      File path.
```

Comment:
Should we provide the file ID?

Reply:
I don't think it is necessary, the room ID includes the file ID. In addition, if we show messages to the user, they don't know what file it is.